Cloudinary/Api.cs

```diff
@@ -77,7 +77,7 @@ static Api()
         /// <param name="parameters">Dictionary of call parameters (can be null)</param>
         /// <param name="file">File to upload (must be null for non-uploading actions)</param>
         /// <returns>HTTP response on call</returns>
-        public HttpWebResponse Call(HttpMethod method, string url, SortedDictionary<string, object> parameters, FileDescription file, Dictionary<string, string> extraHeaders = null)
+        public override object Call(HttpMethod method, string url, SortedDictionary<string, object> parameters, FileDescription file, Dictionary<string, string> extraHeaders = null)
```

Comment:
The method is internal, call it a service method. There are a lot of things going on in it, parameters preparation, their processing, etc. It was initially planned as private, not sure why it became public. Moreover, it is not described in the docs, so SDK users call it at their own risk.
This change of method, of the returned type, allows us to write the whole code into the "Shared". This was the cornerstone for this whole project and it means that new methods described in "Shared", work on both .Net Core and Net Classic, so there's no need to duplicate the code.